hyperlink/test/test_scheme_registration.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from .common import HyperlinkTestCase
+from .._url import register_scheme, URL
+
+
+class TestSchemeRegistration(HyperlinkTestCase):
+
+    def test_register_scheme_basic(self):
+        register_scheme('deltron', uses_netloc=True, default_port=3030)
```

Comment:
Also, what happens if I do:

```python
register_scheme("something")
```

And `uses_netloc` remains `None`?  [It looks like nothing!](https://github.com/python-hyper/hyperlink/blob/master/hyperlink/_url.py#L335-L336)  If that's the case, `uses_netloc` should be a required argument.